src/core/models/aggregate.cc

```diff
@@ -51,7 +51,7 @@ grouping for string columns.
 
 When the input frame has more columns than two,
 a parallel one-pass Ad-Hoc algorithm is employed, see description of
-`Aggregator<T>::group_nd() <https://github.com/h2oai/datatable/blob/main/src/core/models/aggregator.cc>`_
+`Aggregator<T>::group_nd() <https://github.com/h2oai/datatable/blob/main/src/core/models/aggregate.cc#L879>`_
```

Comment:
Nice catch, I guess we can even make it a link like to a particular commit (the last one in this case) `https://github.com/h2oai/datatable/blob/00185546f033cc9d2fca202b516a1720b85581f5/src/core/models/aggregate.cc#L878-L1024` so that if we have a minor change in the source code, we don't need to worry about the docs. If we have a major change, I will update this link.

Reply:
Why not just transfer the description from code into the docs?

Reply:
That description was for the developer use only, so I would prefer to revisit it and make the algo more clear for the general user first.

Reply:
Yeah, I understand that. But since we are mentioning it explicitly in the public docs, then it kinda no longer is "developer only".

Reply:
Yep, that link was a temporary solution... I will make a separate PR to improve the aggragator docs at some point.